courses/machine_learning/deepdive2/image_classification/solutions/3_tf_hub_transfer_learning.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<p>&nbsp;&nbsp;&nbsp;&nbsp;max_delta = 63.0 / 255.0&nbsp;# Change brightness by at most 17.7%</p><p> </p><p>make this all caps constant outside the function</p><p> </p><p>-----</p><p> </p><p>&nbsp;&nbsp;# Pixel values are in range [0,1], convert to [-1,1]</p><p> </p><p>why are is this neccessary? Can you add a comment why</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Ack for the first.</p><p>Good call out for the second. I copied it over from the previous version of the lab, and didn't think much of it. Gave it a try without, and worked just as well, so axed it.</p>